### DISCUSSION TITLE
Doorbell Camera Stream

### DISCUSSION THREAD

**Discussion by Dud3z** [andrei-tatar/node-red-contrib-smartnora#267] [Ideas]
Hi, 

is it possible to implement the camerastream to the doorbell node?
i think some guys here would use this.

This way we could show the stream of a non Google Doorbell.

**Comment by Dud3z** [comment]
Nothing after 2 yrs?

**Reply by andrei-tatar** [reply]
It should be possible, everything that google needs is exposed via NORA.
I didn't personally try it out.

**Reply by andrei-tatar** [reply]
I can look into giving it a shot. Don't think I can manage the time before next year.
